core/modules/image/image.field.inc

```diff
@@ -200,6 +203,26 @@ function image_field_instance_settings_form($field, $instance) {
   return $form;
 }
 
+/**
+ * #element_validate handler for image_field_instance_settings_form().
```

Comment:
If you're still nearby, can we also change this to `Element validate` instead of `#element_validate`?

Reply:
Done